tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Input/JSON/JSONType.swift

```diff
@@ -36,19 +36,21 @@ internal struct JSONObject: JSONType {
         let name: String
         let comment: String?
         let type: JSONType
-        let defaultVaule: DefaultValue?
+        let defaultValue: DefaultValue?
         let isRequired: Bool
         let isReadOnly: Bool
     }
 
     let name: String
     let comment: String?
     let properties: [Property]
+    let additionalProperties: Property?
```

Comment:
I think that we're using the `Property` type here with a wrong purpose. Looking at how "additionalProperties" work, only `Property.type`, `Property.isReadOnly` and `Property.comment` apply to it and we force artificial / uncontrolled values for `name`, `defaultValue` and `isRequired`. Maybe it's worth introducing separate type: `JSONObject.AdditionalProperties(comment:type:isReadOnly)`, WDYT?

Reply:
I think that it's a good point and good idea!